### PR TITLE
WIP: temporary online meeting support for emergencies

### PR DESCRIPTION
**Pull request by FlipperPA**
This is by no means pretty, but it works, and is a good spot to start conversation.

https://www.aasepia.org/?day=any&search=online

I've built in quick and dirty support for online meetings. It takes two fields, `video_conference_url` and `video_conference_dial_in`, and if they are populated, replaces the `Location` column of the table.

I haven't done anything to the Meeting Detail screen (yet), but this is what we needed for the current emergency.

We also have prefixed meeting titles with (SUSPENDED) or (ONLINE ONLY), so we now have four meeting statuses: ACTIVE, SUSPENDED, ONLINE ONLY, and PERMANENTLY INACTIVE.

This works for us for now, and I figure we can continue to work on refining this into something better.

I hope you are staying safe, @joshreisner. Take care.

### CODE REVIEW

File: src/components/table.jsx [hunk 1]
```diff
@@ -21,6 +21,16 @@ export default class Table extends Component {
...
         />
       );
+    } else if (key == 'location' && meeting.video_conference_url) {
+      return (
+        <div className="location">
+          <a href={meeting.video_conference_url} target="_blank">
+            {meeting.video_conference_url}
+          </a>
+          <br />
+          Dial In: {meeting.video_conference_dial_in}
```
Comment (FlipperPA) [inline src/components/table.jsx]: All of our meetings have both of these fields, but this should only be displayed if populated.

**Comment by FlipperPA** [comment]
@joshreisner Ignore this for now. I pushed too much. I'll revise it tomorrow morning, I desperately need some sleep!

File: src/helpers/data.js [hunk 2]
```diff
@@ -264,7 +266,10 @@ export function loadMeetingData(meetings, capabilities) {
 
       //flags
       meeting.flags = settings.flags
-        .filter(type => lookup_type_values.includes(type) && meeting.types.includes(type))
+        .filter(
+          type =>
+            lookup_type_values.includes(type) && meeting.types.includes(type)
+        )
         .sort()
```
Comment (FlipperPA) [inline src/helpers/data.js]: JSPrettify is working. :)

**Comment by joshreisner** [comment]
Hey @FlipperPA I need to take a few steps back from this project, so I can't approve any new merges until I'm less burnt out. Go ahead with what you're doing of course. The only thing I'd say is `video_conference_phone` would be more consistent than `_dial_in` with existing terminology.

**Pull request closed without merging**